permissions/plugin-uleska.yml

```
@@ -0,0 +1,12 @@
+---
+name: "uleska"
+github: "jenkinsci/uleska-plugin"
+paths:
+  - "org/jenkins-ci/plugins/uleska"
```

```suggestion
  - "io/jenkins/plugins/uleska"
```

This is the <groupId> and <artifactId> from your pom with periods replaced with slashes. https://github.com/jenkinsci/uleska-plugin/blob/master/pom.xml#L11